jmclient/jmclient/client_protocol.py

```diff
@@ -686,17 +705,20 @@ def start_reactor(host, port, factory=None, snickerfactory=None, ish=True,
                     jlog.error("Tried 100 ports but cannot listen on any of them. Quitting.")
                     sys.exit(EXIT_FAILURE)
                 port += 1
+        daemon_serving_port = port
+        daemon_serving_host = host
+
         if snickerfactory:
             start_daemon(host, port-1000, sdfactory, usessl,
                                  './ssl/key.pem', './ssl/cert.pem')
             jlog.info("(SNICKER) Listening on port " + str(port-1000))
-    if usessl:
+        if usessl:
         if factory:
             reactor.connectSSL(host, port, factory, ClientContextFactory())
         if snickerfactory:
             reactor.connectSSL(host, port-1000, snickerfactory,
                            ClientContextFactory())
-    else:
+        else:
```

Comment:
Looks like another `IndentationError`.

Reply:
Done in https://github.com/JoinMarket-Org/joinmarket-clientserver/pull/670/commits/5b0bf4ae2abeded214053f8bc54392cdae2897ba